frontend/src/bundles/common/components/player/player.tsx

```diff
@@ -50,11 +50,12 @@ const Player = ({
                 compositionWidth={size.width}
                 compositionHeight={size.height}
                 fps={30}
-                style={
-                    orientation === VideoPreview.LANDSCAPE
+                style={{
+                    maxHeight: '100%',
+                    ...(orientation === VideoPreview.LANDSCAPE
```

Comment:
Please add the `maxHeight: '100%'` property to the `PortraitStyle` and `LandscapeStyle` objects instead of directly adding it in the style props.